packages/appstore/src/components/modals/real-wallets-upgrade/components/title-text/title-text.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Text } from '@deriv/components';
+import './title-text.scss';
+
+const TitleText = ({ children, className }: React.PropsWithChildren<{ className?: string }>) => {
+    return (
+        <Text as='span' className={classNames('title-text', className)} color='prominent' align='center'>
```

Comment:
```suggestion
        <Text className={classNames('title-text', className)} color='prominent' align='center'>
```
as='span' is a default prop and you don't need to pass it